src/redux/carSlice.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable no-param-reassign */
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+export const fetchCars = createAsyncThunk('cars/fetchCars', async () => {
+  const response = await axios.get('/api/cars');
```

Comment:
Please fix the url + try to add the base url `(http:localhost:3000/api/v1/cars)`